### ISSUE TITLE
Focusing TextField on web throws LateInitializationError error

### ISSUE THREAD

**Issue by cedvdb** [flutter/flutter#87343]
reopening [original issue
](https://github.com/flutter/flutter/issues/83618) since it is now closed.

## Steps to Reproduce

The original poster posted this snippet which still throws an error on dart pad

 `Error: LateInitializationError: Local 'command' has not been initialized.`

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.title}) : super(key: key);

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: TextField(
          keyboardType: TextInputType.emailAddress,
          autofillHints: const <String>[
            AutofillHints.email,
          ],
        ),
      ),
    );
  }
}
```

and another example is in the dev branch example here https://github.com/cedvdb/phone_form_field/tree/dev

**Comment by hellochenwang** [comment]
is there a fix for this issue?

**Comment by vishnuagbly** [comment]
I am facing the same issue, did anyone find a fix?

**Comment by TahaTesser** [comment]
Hi @cedvdb 
This has been fixed, currently, the beta channel has no such issue when focusing TextField

Can you please upgrade to the latest  `beta` and try again
Execute the following commands to upgrade
`flutter channel beta`
`flutter upgrade --force`
`flutter doctor -v`


Closing as duplicate of https://github.com/flutter/flutter/issues/83618

**Comment by cedvdb** [comment]
@TahaTesser How can we know it's in the beta channel for future issues ? Because I linked to that issue in the original post without knowing it was in the beta channel.

The merge was back in may.

**Comment by TahaTesser** [comment]
@cedvdb 
The fix exist from 2.3, currently stable is `2.2.3` and beta is `2.4`, upgrading to the beta channel, you should have no issue
